docs/docs/when-to-use-use.md

```diff
@@ -0,0 +1,232 @@
+# When to use `use`?
+
+`use` connects resources to _JavaScript_ contexts.
```

Comment:
It would be helpful to explain right away why the connection is important/neccessary

Reply:
explained below, lemme know what you think!